hocr-simplify

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+# change level of typesetting and/or remove properties
+# to create a simplified hocr-version
+
+from __future__ import print_function
+import argparse
+import re
+import sys
+import os
+
+from lxml import etree, html
+
+parser = argparse.ArgumentParser(
+    description=('change level of typesetting and/or'
+                 'remove properties to create'
+                 'a simplified hocr-version'))
+properties = ['baseline', 'bbox', 'cflow', 'cuts', 'hardbreak', 'image',
+              'imagemd5', 'lpageno', 'ppageno', 'nlp', 'order', 'poly',
+              'scan_res', 'textangle', 'x_booxes', 'x_font', 'x_fsize',
+              'x_confs', 'x_scanner', 'x_source', 'x_wconf']
+
```

Comment:
It would be nice to have also an option to delete `id` and/or `dir` parameter, but they are on their own.

Reply:
Removing attributes is now implemented